content/applications/hr/fleet/service.rst

```diff
@@ -86,61 +298,219 @@ clicked, sort the data in different ways.
+
+The number of repairs for each type of service appears in the far-right of each Kanban column
+header.
+
+The status of any scheduled activities appears in the color-coded bar beneath each Kanban column
+title. Repairs with activities scheduled in the future appear green, activities due today appear
+yellow, overdue activities appear red, and repairs with no activities scheduled appear gray.
+
+Each Kanban card displays a color-coded :guilabel:`(activity icon)`, indicating what type of
+activity is scheduled, and the status. For example, a scheduled phone call displays a phone icon.
+The status colors correspond to the colors in the status bar. Click on an :guilabel:`(activity
+icon)` to view the details of the activity.
+
+The length of the color bar is proportionate to the number of corresponding activities with that
+status. Hover over a color section to reveal the number of service records in that specific
+grouping.
+
+.. image:: service/kanban.png
+   :align: center
+   :alt: The service records presented in a Kanban view.
+
+To view **only** the records with a specific status, click the desired :guilabel:`color bar`
+section. The background color for the column changes to a pale hue of the same color (either green,
+yellow, red, or gray), and the color bar appears striped instead of solid. **Only** repairs and
+services with the selected status appear in the column.
+
+.. image:: service/kanban-status.png
+   :align: center
+   :alt: The oil change service records showing only repairs with past-due activities.
+
+.. _service/schedule-activity:
+
+Schedule activities
+*******************
+
+To schedule an activity for a repair or service from the Kanban view, click the :guilabel:`activity
+icon` in the lower-right corner of the service record, and click :guilabel:`+ Schedule an activity`.
+A :guilabel:`Schedule Activity` pop-up window appears.
+
+Enter the following information on the form:
+
+- :guilabel:`Activity Type`: using the drop-down menu, select the activity being scheduled. The
+  default options are :guilabel:`Email`, :guilabel:`Call`, :guilabel:`Meeting`, :guilabel:`To-Do`,
+  and :guilabel:`Upload Document`.
+- :guilabel:`Summary`: enter a short description of the activity, such as `Schedule oil change`.
+- :guilabel:`Due Date`: using the calendar popover, select the date the activity must be completed.
+  Using the :guilabel:`< (left)` and :guilabel:`> (right)` arrow icons, navigate to the desired
+  month, then click on the :guilabel:`date` to select it.
+- :guilabel:`Assigned to`: using the drop-down menu, select the user responsible for the activity.
+- :guilabel:`Notes`: add any notes or details in the blank area in the bottom-half of the form.
+
+When the :guilabel:`Schedule Activity` is completed, click :guilabel:`Schedule` to schedule the
+activity, or click :guilabel:`Done & Schedule Next` to schedule the current activity and schedule
+another activity for the same repair.
+
```

Comment:
I am torn- because I do like that they have everything they need here. But there is a place in the activities doc where they do show adding an activity from the kanban view (and now I want to update that doc because it is so old!) Ugh.. I honestly don't know how I feel! What a cop out, I know. Although when I look at the doc, it is such a small section, I feel like I want to keep it more so it's a "one and done" or an "everything you need" doc.